components/autofill/android/src/test/java/mozilla/appservices/autofill/AutofillTest.kt

```diff
@@ -17,7 +16,6 @@ class AutofillTest {
     val dbFolder = TemporaryFolder()
 
     fun createTestStore(): Store {
-        Megazord.init()
```

Comment:
`RustComponentsInitializer` not needed here? I did not look into the details but autofill requires `jwcrypto`, which most of the times need to be initialized.

Reply:
I was surprised by this too - I ran the tests locally too without them failing. After reading the called code, we don't execute any crypto bits in there. It only touches the sync engine registration.